content/en/cosign/overview.md

```diff
@@ -98,18 +148,9 @@ Enter password for private key:
 Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79be4def8.sig
 ```
 
-### Other Formats
+## Other Formats
 
 Cosign is useful not only for blobs, containers, and container-related artifacts; it can also be used for other file types.
 
-To learn how to sign SBOMs, WASM modules, Tekton bundles and more, review [Signing Other Types](/cosign/other_types/). For more information about blobs, review [Working with Blobs](/cosign/working_with_blobs/).
-
-### SCM Integration
```

Comment:
Can we leave these two sections in this quickstart for now? I think we can figure out a better place for them later, but I don't think they should be in verify.md.

Let's also update header to ## rather than ###